packages/calcite-components/src/components/date-picker-month/date-picker-month.scss

```diff
@@ -7,13 +7,11 @@
  * @prop --calcite-date-picker-month-day-background-color-selected: Specifies the background color of day range when selected.
  * @prop --calcite-date-picker-month-day-background-color: Specifies the background color of day.
  * @prop --calcite-date-picker-month-day-corner-radius: Specifies the border radius of day.
- * @prop --calcite-date-picker-month-day-font-size: Specifies the font size of day.
  * @prop --calcite-date-picker-month-day-range-background-color-hovered: Specifies the background color of day range when hovered.
  * @prop --calcite-date-picker-month-day-range-background-color-selected: Specifies the background color of day range when selected.
  * @prop --calcite-date-picker-month-day-text-color-hover: Specifies the text color of day when hovered.
  * @prop --calcite-date-picker-month-day-text-color: Specifies the text color of day.
  * @prop --calcite-date-picker-month-day-text-selected: Specifies the text color of day when selected.
- * @prop --calcite-date-picker-month-week-headers-border-color: Specifies the border color of week headers.
  * @prop --calcite-date-picker-month-week-headers-text-color: Specifies the text color of week headers.
```

Comment:
is there a reason we need different variables for month, day, week? Shouldn't all date picker tokens fallback to the same `--calcite-date-picker-` tokens. Especially since some of these components are internal only?